### DISCUSSION TITLE
ModuleNotFoundError: No module named 'rospkg'

### DISCUSSION THREAD

**Discussion by ElaineYao** [USC-ACTLab/crazyswarm#629] [Q&A]
Hi,

I am following the [instructions](https://crazyswarm.readthedocs.io/en/latest/installation.html) to install physical robots and simulations. 
When running `$CSW_PYTHON -m pytest`, it gave me the following error:
```
E   ModuleNotFoundError: No module named 'rospkg'

/opt/ros/noetic/lib/python3/dist-packages/roslib/launcher.py:42: ModuleNotFoundError
```
I tried `sudo apt-get install python3-rospkg` but the error still exists.

**Comment by jpreiss** [comment]
you may have forgotten to source setup.bash.

**Reply by ElaineYao** [reply]
Solved. Referring to [this](https://githubhot.com/repo/ros-infrastructure/rospkg/issues/226)

> the most common reason I've seen missing system modules like this results from the use of a non-default Python interpreter. 
> Please run which python3 to determine where your interpreter lives. If it doesn't live at /usr/bin/python3, then you have two choices:
